modules/apps/data-engine/data-engine-service/src/main/java/com/liferay/data/engine/internal/exportimport/data/handler/DEDataDefinitionFieldLinkStagedModelDataHandler.java

```diff
@@ -161,8 +162,15 @@ protected void doImportStagedModel(
 		importedDEDataDefinitionFieldLink.setDdmStructureId(ddmStructureId);
 
 		if (className.equals(DDMStructureLayout.class.getName())) {
+			long linkedStructureId = GetterUtil.getLong(
+				deDataDefinitionFieldLinkElement.attributeValue(
+					"linked-structure-id"));
```

Comment:
@mbowerman I think we could rename it to something like `layout-ddm-structure-id`, to make it clearer that this ID is related to the `DDMStructureLayout` and not related to the `ddmStructureId` column in the `DEDataDefinitionFieldLink` table. What do you think?

Reply:
@carolmariaabb Done